### PR TITLE
Fix compatibility with React Native 0.46 final release

### PR DESCRIPTION
**Pull request by petejkim**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ const process = require('process')
 const TSCONFIG_PATH = process.env.TSCONFIG_PATH
 var upstreamTransformer = null
 try {
-  upstreamTransformer = require('metro-bundler/src/transformer')
+  // handle RN 0.46
+  upstreamTransformer = require('metro-bundler/build/transformer')
```
Comment (ds300) [inline index.js]: Hi, thanks for the contribution! This line is the only change I need and want to keep from your PR. Can you revert the others please?

Comment (petejkim) [inline index.js]: sure

**Comment by petejkim** [comment]
Updated

**Comment by ds300** [comment]
Cool thanks. I'm about to go to bed. Should get around to testing this at some point tomorrow.

**Comment by iRoachie** [comment]
@petejkim Does this work for you? i changed it manually but it now shows, No bundle URL present

**Comment by petejkim** [comment]
This requires the latest metro-bundler react-native depends on (^0.7.4)

If you see messages like `Cannot find module 'react-native/packager/transformer'`, you should do `rm yarn.lock package-lock.json && rm -rf node_modules && yarn install`.

**Comment by iRoachie** [comment]
Yea @petejkim It's working. Other thing now is another transform issue with babel https://github.com/facebook/react-native/issues/14866

**Comment by petejkim** [comment]
I didn't have that issue, but I was upgrading from 0.45.1 to 0.46, I believe new project generation is broken

**Comment by ds300** [comment]
Nice thanks. I was getting the same issue as @iRoachie with a brand new project and 0.46.0, but things worked after I restarted the packager (with this change) and rebuilt the app.

**Comment by ds300** [comment]
image

**Pull request merged**